### DISCUSSION TITLE
Adding to Winget (Window Package Manager)

### DISCUSSION THREAD

**Discussion by arfx** [ggarra13/mrv2#216] [Ideas]
Hey I just noticed that I'm way behind the current version.
Can you add your tool to winget to see when an update is available and install this with the Window Package Manager.

https://learn.microsoft.com/en-us/windows/package-manager/package/

**Comment by ggarra13** [comment]
I don't have it set up for winget.  However, if you have a more or less recent version (> 1.0), you can do:

Help->Update mrv2

and it will download the latest version of mrv2 from github and proceed to install it.  It is not as elegant as I would like, but it should work.

Please report if it does not work for you or if you run into any issues.

**Comment by ggarra13** [comment]
Oh.  I just noticed in the update script for Windows that it will expect the path to be:

`C:/Program Files/mrv2 {version}/bin/mrv2.exe`

and recent versions are installing it in:

`C:/Program Files/mrv2-{version}/bin/mrv2.exe`

I'll fix the update script for the next release so that it checks on both locations.

**Comment by ggarra13** [comment]
I investigated winget and unfortunately, it is too limited.  Installers like mrv2's installer which allow setting what components to install and what file associations to set cannot be automated with winget.
